notebooks/01_prepare_data/wikidata.ipynb

```diff
@@ -0,0 +1,369 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set the environment path to find Recommenders\n",
+    "import sys\n",
+    "sys.path.append(\"../../\")\n",
+    "\n",
+    "import pandas as pd\n",
+    "from reco_utils.dataset.wikidata import (\n",
+    "    find_wikidataID,\n",
+    "    query_entity_links,\n",
+    "    read_linked_entities,\n",
+    "    query_entity_description\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Create a KG from linked entities in Wikidata from a list"
```

Comment:
I expect some text explanations at the beginning of the notebook about knowledge graph, basic concepts like entities, Wikidata, etc.

Reply:
Added the info in the first cell :)